slides/all-templates/index.ftd

```diff
@@ -0,0 +1,137 @@
+
+-- component featured-template:
+caption title:
+ftd.image-src thumbnail:
+ftd.image-src owner_avatar:
+string owner_name:
+string cta-text:
+boolean $pop-up: false
+
+-- ftd.row:
+width: fill-container
+padding-vertical.px: 24
+padding-horizontal.px: 40
+spacing.fixed.px: 24
+
+-- ftd.column:
+width.fixed.px: 1014
+spacing.fixed.px: 36
+
+-- ftd.image:
+src: $featured-template.thumbnail
+width: fill-container
+height.fixed.px: 570
+border-radius.px: 12
+fit: cover
+
+-- end: ftd.column
+
+-- ftd.column:
+width.fixed.px: 322
+spacing.fixed.px: 16
+
+-- ds.heading-medium: $featured-template.title
+
+-- ftd.row:
+spacing.fixed.px: 8
+align-content: center
+padding-vertical.px: 8
+
+-- ftd.image:
+src: $featured-template.owner_avatar
+width.fixed.px: 32
+
+-- ftd.text: $featured-template.owner_name
+role: $inherited.types.button-small
+color: $inherited.colors.text-strong
+
+-- end: ftd.row
+
+-- ftd.text: $featured-template.cta-text
+background.solid: $inherited.colors.accent.primary
+width: fill-container
+text-align: center
+padding-vertical.px: 16
+padding-horizontal.px: 18
+border-radius.px: 8
+color: $inherited.colors.text-strong
+$on-click$: $ftd.set-bool($a = $featured-template.pop-up, v= true)
+
+-- end: ftd.column
+
+-- ftd.column:
+if: { featured-template.pop-up}
+width.fixed.px: 370
+min-height.fixed.px: 442
+padding.px: 24
+border-radius.px: 8
+background.solid: $inherited.colors.background.step-1
+spacing.fixed.px: 24
+z-index: 2
+anchor: parent
+align-self: center
+left.px: 535
+color: $inherited.colors.text-strong
+
+-- ftd.row:
+width: fill-container
+spacing: space-between
+
+-- ds.heading-small: Create new project
+
+-- ftd.image:
+src: $assets.files.slides.assets.cross.svg
+width.fixed.px: 24
+align-self: center
+$on-click$: $ftd.set-bool($a = $featured-template.pop-up, v= false)
+
+-- end: ftd.row
+
+-- ftd.column:
+spacing.fixed.px: 8
+align-self: center
+
+-- ftd.image:
+src: $featured-template.thumbnail
+width.fixed.px: 200
+border-radius.px: 4
+fit: cover
+
+-- ds.copy-small: 
+text: $featured-template.title
+
+-- end: ftd.column
+
+-- ftd.column:
+width: fill-container
+spacing.fixed.px: 8
+
+-- ftd.text: Project name
+role: $inherited.types.button-small
+
+-- ftd.text-input:
```

Comment:
No. Every thing in this file is `ftd.*`.  I need strong explanation for every `ftd.*` usage. Everything must be in design-system package.